### PR TITLE
restore mixs, mixc and mixcol binaries

### PR DESCRIPTION
**Pull request by mandarjog**
**Release note**:

```release-note
NONE
```

---
This change is [Reviewable](https://reviewable.io/reviews/istio/mixer/1385)

### CODE REVIEW

File: cmd/client/BUILD [hunk 1]
```diff
@@ -17,6 +12,6 @@ go_library(
 )
 
 go_binary(
-    name = "client",
```
Comment (jmuk) [inline cmd/client/BUILD]: This isn't working well with gazelle, that's why @kyessenov made this.

Please mark `# gazelle:ignore` at the top of this file (and other BUILD files in `cmd`).

**Comment by mandarjog** [comment]
Added `gazelle:ignore` and confirmed by running `bin/gazelle`.
It does not change the files. PTAL. @jmuk @guptasu

**Pull request merged**